### DISCUSSION TITLE
will this work if they cut networks from international access? (intranet)

### DISCUSSION THREAD

**Discussion by amihos** [WeAreMahsaAmini/FreeInternet#31] [General]

**Comment by farnoodma** [comment]
@amihos they never cut out the internet access to outside of the country completely (e.g. data centers will still have access, so if you have a server in one of the data centers, you'll have access), however, in case they do, no this won't work. I'm afraid in that scenario, no solution works unless it is something like Starlink.

**Comment by CyborgAnderson** [comment]
As long as authority doesn't use the domain whitelist mode, theoretically v2ray like method is the most advanced way to bypass the block, it encrypts the internet traffic to the real https data, which is impossible to censor.
Invented by Chinese, who face the most strict internet censorship in the planet, of course except North Korea which is using the LAN

**Comment by amihos** [comment]
I need to explain my question. My point is: is it enough to only set dns record and ask arvan cloud to resolve the domain? Will arvan cloud not realize that this is being used for a vpn purpose? What is the difference between using this method to first set a local Iran vps and then create a tunnel to a foreign vps?

**Comment by CyborgAnderson** [comment]
1.What is needed actually is the CDN service of Arvancloud  
CDN service is used for 
-    hide the real IP of the proxy server 
-    suppose the server IP is blocked in Iran, CDN service could make it accessible 
2. The internet traffic is encrypted to real https traffic, your behavior in front of the censorship should be like: you are browsing a innocent website, comparing to traditional VPN protocols, it's harder to detect and block because there are countless https connections on the internet.
(FreeWorld) <=====> VPS outside of Iran            <==HTTPS==>  CDN server network <==HTTPS==> USER
(FreeWorld) <=====> VPN server outside of Iran        <===Traditional VPN protocols ====>               USER 
3. Of course they can always find you if they want, what matters is the cost and your CDN provider is reliable or not, in China they use Cloudflare, another CDN provider who is not so close to Chinese authorities, since Iran government is actually using the old version of Chinese censorship techniques, it's perfect to using Chinese anti-censor techniques to confront it
